code/modules/cargo/packs.dm

```diff
@@ -800,6 +799,14 @@
 	crate_type = /obj/structure/closet/crate/secure
 	dangerous = TRUE
 
+/datum/supply_pack/science/research
+	name = "Machine Prototype Crate"
+	cost = 5000
```

Comment:
5000 = 50 old points, I think. Pretty sure cargo can buy this just with the cash from selling starting crates